src/patterns/organisms/footer/footer.hbs

```diff
@@ -0,0 +1,66 @@
+---
+name: Footer
+description: |
+  A page footer organism, containing common links to compliment primary navigation and an (optional) page language switcher.
+notes: |
+  - The example above shows an embedded Language Switcher component inside the Footer organism.
+  - Inline JavaScript in this example is for demo purposes only. Use external files in production code.
+---
+
+<footer class="mzp-c-footer mzp-has-lang-switcher">
+  <div class="mzp-l-content">
+    <nav class="mzp-c-footer-primary">
+      <div class="mzp-c-footer-primary-logo"><a href="https://www.mozilla.org/">Mozilla</a></div>
+      <section class="mzp-c-footer-section">
+        <h5>
+          <a href="https://www.mozilla.org/">Mozilla</a>
+        </h5>
+        <ul class="mzp-c-footer-links-mozilla">
```

Comment:
Are these lists (this one and the `mzp-c-footer-links-firefox` one below) distinctive enough to get specific component classes? It seems like they're both common lists and we could use a class on the parent section to distinguish that one is Mozilla and one is Firefox (and even that distinction may not be necessary).

Reply:
Neither are referenced in the CSS, so I think we're fine to remove these entirely 👍